### PR TITLE
Niloofar Sadeghi / Fix broken terms and condition link for "zh-tw" and "zh-cn" language

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — DISMISSED** [review]

**Comment by sean-binary** [comment]
Sorry about this but there was some refactor on the code pushed to master that caused conflicts. Could you please pull the latest master and help to resolve the conflicts?

**Comment by niloo-fs** [comment]
> Sorry about this but there was some refactor on the code pushed to master that caused conflicts. Could you please pull the latest master and help to resolve the conflicts?

Hi @sean-binary , no worries I resolved the conflict. I'll push it with the changes.
I'm still working on refactoring all navigation, I need to test them and then push them here

**Comment by niloo-fs** [comment]
> Sorry about this but there was some refactor on the code pushed to master that caused conflicts. Could you please pull the latest master and help to resolve the conflicts?
I resolved the conflict

File: src/common/constants.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ export const deriv_com_app_id = 16929
 export const deriv_me_app_id = 1411
 export const deriv_be_app_id = 30767
 
+export const supported_domains = [deriv_com_url, deriv_me_url]
+export const domains = [...supported_domains, 'localhost', 'deriv-com-git-fork'] //deriv-com-git-fork for vercel server, localhost - for developer mode
+
 const domain_url_pair = {
     [deriv_com_url]: deriv_com_url,
     [deriv_me_url]: deriv_me_url,
```
Comment (sean-binary) [inline src/common/constants.ts:17]: Could we consolidate these? The `domain_url_pair` was actually to replace `supported_domains`.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**